lib/samples/create_buffers_around_points/create_buffers_around_points.dart

```diff
@@ -0,0 +1,410 @@
+//
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'dart:math';
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/utils/sample_state_support.dart';
+import 'package:flutter/material.dart';
+
+class CreateBuffersAroundPoints extends StatefulWidget {
+  const CreateBuffersAroundPoints({super.key});
+
+  @override
+  State<CreateBuffersAroundPoints> createState() =>
+      _CreateBuffersAroundPointsState();
+}
+
+class _CreateBuffersAroundPointsState extends State<CreateBuffersAroundPoints>
+    with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+
+  // A polygon that represents the valid area of use for the spatial reference.
+  late Polygon _boundaryPolygon;
+
+  // List of tap points.
+  final _bufferPoints = <PointAndRadius>[];
+
+  // Current status of the buffer.
+  var _status = Status.addPoints;
+
+  // Buffer radius.
+  var _bufferRadius = 100.0;
+
+  // Union status.
+  var _shouldUnion = false;
+
+  // A flag for when the settings bottom sheet is visible.
+  var _showSettings = false;
+
+  // Define the graphics overlays.
+  final bufferGraphicsOverlay = GraphicsOverlay();
+  final tapPointGraphicsOverlay = GraphicsOverlay();
+
+  // Fill symbols for buffer and tap points.
+  final bufferFillSymbol = SimpleFillSymbol();
+  final tapPointSymbol = SimpleMarkerSymbol();
+
+  final statePlanNorthCentralTexasSpatialReference =
+      SpatialReference(wkid: 32038);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: true,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                // Display a banner with instructions at the top.
+                SafeArea(
```

Comment:
other places we stack this "on top" (z-order) of the map. So it's in the Stack's children.

Reply:
Changed it to the stack children.